src/cdk/testing/tests/BUILD.bazel

```diff
@@ -11,6 +11,7 @@ ng_module(
             "**/*.spec.ts",
             "**/*.spec.d.ts",
             "harnesses/**",
+            "cross-environment-specs.ts",
```

Comment:
Can you rename this file to `cross-environment.spec.ts`? That way it will be automatically excluded by the globs above